### ISSUE TITLE
Change `@eventespresso/i18n` to `@wordpress/i18n` in webpack config.

### ISSUE THREAD

**Issue by nerrad** [eventespresso/event-espresso-core#1067]
Once we bump the minimum WP required in EE core (or any of our add-ons) we should change the webpack config to have the `@eventespresso/i18n` external point to `wp.i18n` instead of `eejs.i18n` and then remove the wrapper we generate for `eejs.18n`.

Then we can start using `@wordpress/i18n` in our imports going forward.

The only reason we had `@eventespresso/i18n` to begin with was because we were supporting versions of WP that do not have `@wordpress/i18n` bundled and this allowed us to more easily support back-compat.

**Comment by joshfeck** [comment]
> Once we bump the minimum WP required in EE core

Which version? 5.0, 5.2?

**Comment by nerrad** [comment]
I put 5.0 because that's the minimum version of WP core needed to make this change.  In reality though, it'd probably be implemented when we bump min required version to 5.2.  Does that make sense?

**Comment by joshfeck** [comment]
Yes. It makes sense.

If you want to start on this now, then part of the work on this ticket could include bumping the minimum WP version to 5.0. Otherwise this needs to be put on hold until some time after WP 5.2 is released.

**Comment by nerrad** [comment]
I don't think it needs to be done before anything else that bumps the min version because what we are using now is okay.  It's more a "okay not that we require WP 5.x+ we can get this cleaned up".

**Comment by joshfeck** [comment]
Fair enough. We'll wait until we require WP 5.x before getting started on this.

**Comment by tn3rb** [comment]
no longer applicable
